lib/bibliothecary/parsers/maven.rb

```diff
@@ -109,6 +115,20 @@ def self.parse_gradle_resolved(file_contents)
         end.compact.uniq {|item| [item[:name], item[:requirement], item[:type]]}
       end
 
+      def self.parse_maven_resolved(file_contents)
+        file_contents.split("\n").map do |line|
+          type_match = MAVEN_DEP_REGEXT.match(line)
```

Comment:
Can we just split on `:` here? It might be easier to reason about than the regex

Reply:
Yeah that sounds more reasonable.